health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -791,6 +791,11 @@ export const UICustomizations = {
       const tabId = url.tabId || "0"; // Default to '0' if tabId is undefined
       data.body.PlanConfigurationSearchCriteria.status = dic[String(tabId)];
       cleanObject(data.body.PlanConfigurationSearchCriteria);
+      if(data.body.PlanConfigurationSearchCriteria.status && data.body.PlanConfigurationSearchCriteria.status[0]==="RESOURCE_ESTIMATIONS_APPROVED"){
+        delete data.body.PlanConfigurationSearchCriteria.userUuid
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)